flutter_communication_bridge/pubspec.yaml

```diff
@@ -23,8 +23,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  webview_flutter: ^1.0.7
-
+  webview_flutter: ^3.0.0
```

Comment:
@Mr-MaNia7  - Why not upgrade to the latest version?

Reply:
The WebView widget you have used have been depreciated. When migrating the package from 3.0 to 4.0 as found in the official documentation page:
> The WebView class has been removed and its functionality has been split into WebViewController and WebViewWidget.

And the last version to support `WebView` widget is version `^3.0.0`.
More info can be found [here](https://pub.dev/packages/webview_flutter#migrating-from-30-to-40). Thanks.

Reply:
@Mr-MaNia7  - You can upgrade to version 4.0.7 which uses a WebViewController.